contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -6331,6 +6331,14 @@ bbf_ExecDropStmt(DropStmt *stmt)
 			if (schema_name != NULL)
 				logicalschema = get_logical_schema_name(schema_name, true);
 
+			/* Restrict dropping of specific procedures */
+			if (stmt->removeType == OBJECT_PROCEDURE && major_name && pg_strncasecmp(major_name, "xp_", 3) == 0)
```

Comment:
Can there be user defined procedures whose names start with "xp_*"? If yes, those should be allowed to drop.

Reply:
Right, there can be such cases. I'll update the logic for this as there are some more stored procedures which should be restricted from being droppped.